src/Components/TopContentShow.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import { Menubar } from "primereact/menubar";
+import { MenuItem } from "primereact/menuitem";
+import { useNavigate } from "react-router-dom";
+import { Divider } from "primereact/divider";
+import { InputText } from "primereact/inputtext";
+import { Button } from "primereact/button";
+import { Card } from "primereact/card";
+
+interface Product {
```

Comment:
we can make new folder /src/models/model_name (Product here) to more organize your code.